document/chapters/04-simd_bvh.tex

```diff
@@ -261,34 +264,37 @@ \subsection{Transport Tests}
+the nested cubes model, it is likely that the number of surfaces relative to the
+number of triangles in their representation is high enough to allow EmDAG-MCNP
+to overtake MCNP's CSG calculations. This is a good demonstration of how CSG
+implementations suffer from the lack of a spatial search component when creating
+volumes from Boolean combination of surfaces as mentioned in Section
+\ref{subsec:csg}.
 
 The results of the single-volume test cases for native MCNP differ slightly from
-the tally results from the DAGMC-based systems, which are the same. This is not
-surprising as DAGMC is known to report statistically similar results to those of
-native MCNP. Comparisons of DAGMC to native codes are not the concern of this
-study, only a comparison of the values returned by EmDAG in comparison to DAGMC
-is considered. Differences in the tally results between DAG-MCNP and EmDAG-MCNP
-are present only in the nested spheres transport model. There is a small
-difference in the flux tally for the outermost volume as can be seen in Table
-\ref{nestedspheres} of Appendix \ref{ch:appendix-b}. By examining the number of
-particle tracks in each cell, it was determined via debugging values that this
-discrepancy is caused by a single particle ending in EmDAG-MCNP near a surface
-of cell 2 while in DAG-MCNP the particle crosses into cell 3 before abruptly
-terminating though it still contributing slightly to the tally in cell 3. It is
-believed that this difference in tally result is the result of a systematic
-difference between Embree and MOAB's ray fire conventionality rather than being
-result of the double to single floating point conversion of the model that
-occurs when using EmDAG - though this conversion in precision is shown to be
-problematic in other ways in Section \ref{sec:emdag_limitations}.
+the results from the DAGMC-based systems, which match each other exactly. This
+is not surprising as DAG-MCNP is known to report statistically similar, but not
+exactly the same, results as native MCNP. Comparisons of DAG-MCNP to native
+codes are not the concern of this study, only a comparison of the values
+returned by EmDAG-MCNP in comparison to DAG-MCNP is considered. Differences in
+the tally results between DAG-MCNP and EmDAG-MCNP are present only in the nested
+spheres transport model. There is a small difference in the flux tally for the
+outermost volume as can be seen in Table \ref{nestedspheres} of Appendix
+\ref{ch:appendix-b}. By examining the number of particle tracks in each cell, it
+was determined via debugging values that this discrepancy is caused by a single
+particle ending in EmDAG-MCNP near a surface of cell 2 while in DAG-MCNP the
+particle crosses into cell 3 before abruptly terminating though it still
+contributing slightly to the tally in cell 3. It is believed that this
+difference in tally result is the result of a systematic difference between
+Embree and MOAB's ray fire conventionality rather than being result of the
```

Comment:
Is “conventionality” a word?

Reply:
Unlike words I've made up in the past, it is! 

https://www.merriam-webster.com/dictionary/conventionality

Reply:
so now I'm not sure it's the right word...  do you mean "convention"? or something else/  I guess this description doesn't really communicate to me what really causes any differences.

Reply:
Regardless of which word is used, I think you're right - it doesn't communicate the differences well. It's worth explaining this in more detail. 

Also, "it is believed" is soft wording here. I tracked this down in the particle history and know why the problem is there.